extensions/wikia/PageHeader/styles/index.scss

```diff
@@ -20,6 +23,10 @@
 		word-wrap: break-word;
 	}
 
+	&__dropdown-padding {
```

Comment:
I'm not sure about this kind of class' naming convention just to overwrite one particular property - the class should not be dependant on the styling properties it defines. I would rather recommend changing it to something like `.page-header__categories-dropdown` .